1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,176 @@
+Feature: User edits Project
+    In order to document a project 
+    As a logged in User
+    I want to edit existing and create new projects
+
+    @xxx
+    Scenario Outline: User sees the Project Wizard
+        Given User opens My Projects
+        When they click Create New Project Button
+        Then the User is the Project Owner
+        And they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        # Save button is removed as Project API demands a PM
```

Comment:
Kanskje i overkant kompleks logikk for lagre-knapp nå?

Description tab: Aldri mulighet til å lagre
Participants tab: Alle får enten `Save draft` (hvis draft) eller `Update published project` (hvis publisert)
Financing tab: Alle får `Save and Present`

Reply:
Diskutert